test/util/ChartUtils/getDomainOfDataByKey.spec.tsx

```diff
@@ -0,0 +1,169 @@
+import { getDomainOfDataByKey } from '../../../src/util/ChartUtils';
+
+describe('getDomainOfDataByKey', () => {
```

Comment:
Just to iterate: This is a very high value contribution, thanks a ton. Improving our overall confidence to change things is one of the most important things that does not itself map to an issue or specific feature request.